### ISSUE TITLE
Is there a pytorch pre-training model for LiteFlowNet? I found it very memory intensive when I applied draft, so I need a light weight optical flow estimation network

### ISSUE THREAD

**Issue by izhimahu** [hmorimitsu/ptlflow#33]

**Comment by hmorimitsu** [comment]
I am sorry, I am not sure I understand your problem. LiteFlowNet and its weights are provided in this repository. Is this what you need, or do you mean something different?

**Comment by izhimahu** [comment]
> I am sorry, I am not sure I understand your problem. LiteFlowNet and its weights are provided in this repository. Is this what you need, or do you mean something different?

![image](https://user-images.githubusercontent.com/66286551/151483976-2db4d153-f95d-4f83-821c-ed724d1a5626.png)
These are caffe models, is there a pytorch trained model?

**Comment by hmorimitsu** [comment]
Maybe it is a bit confusing, but this name in the second column is the name of the original checkpoint that I used to generate my own version (the name in the first column). The original LiteFlownet was released in Caffe, but I converted it into Pytorch. Therefore, the checkpoint that you will get when using PTLFlow will be the Pytorch version (liteflownet3-....ckpt).

I hope that helps you. If not, just let me know.
